git_version/templatetags/git_tags.py

```diff
@@ -5,11 +5,15 @@
 
 @register.simple_tag
 def current_commit(path):
-    repo = Repo(path)
-    commit = repo.commit()
-    chash = commit.hexsha
-    repo.__del__()
-    return chash
+    try:
+        repo = Repo(path)
+        commit = repo.commit()
+        chash = commit.hexsha
+        repo.__del__()
+        return chash
+    except Exception:
+        print(f"git_version - trouble with repo path {path}")
```

Comment:
```suggestion
        logger.debug(f"Failed to find current commit for {path}")
```
Can you replace the `print` statement with a logger call as described in <https://docs.djangoproject.com/en/3.2/topics/logging/#using-logging>? That's probably more appropriate for something published as a library.

Just for the sake of completeness, this would also need
```py
import logging
logger = logging.getLogger(__name__)
```
somewhere at the start of the file